### PR TITLE
syncthingtray: Add version 1.1.1

### PR DESCRIPTION
**Pull request by yan12125**

### CODE REVIEW

File: bucket/syncthingtray.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            "32bit": {
+                "url": "https://github.com/Martchus/syncthingtray/releases/download/v$version/syncthingtray-$version-i686-w64-mingw32.exe.zip"
+            },
+            "64bit": {
+                "url": "https://github.com/Martchus/syncthingtray/releases/download/v$version/syncthingtray-$version-x86_64-w64-mingw32.exe.zip"
+            }
+        }
+    },
+    "checkver": {
+        "github": "https://github.com/Martchus/syncthingtray"
```
Comment (Ash258) [inline bucket/syncthingtray.json:43]:
```suggestion
```

File: bucket/syncthingtray.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            "syncthingtray"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "syncthingtray.exe",
+            "Syncthing Tray"
+        ]
+    ],
+    "autoupdate": {
```
Comment (Ash258) [inline bucket/syncthingtray.json:32]:
```suggestion
    "checkver": "github",
    "autoupdate": {
```

File: bucket/syncthingtray.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            "hash": "85c16af32fcafacc7e8d37609f0bdfb3b038e922728d51577083f47f9bbb3565"
+        }
+    },
+    "pre_install": "Move-Item \"$dir\\syncthingtray-*.exe\" \"$dir\\syncthingtray.exe\"",
+    "bin": [
+        [
+            "syncthingtray.exe",
+            "syncthingtray"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/syncthingtray.json:25]:
```suggestion
    "bin": "syncthingtray.exe",
```

File: bucket/syncthingtray.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
+{
+    "version": "1.1.1",
+    "homepage": "https://github.com/Martchus/syncthingtray",
+    "description": "Tray application for Syncthing",
```
Comment (Ash258) [inline bucket/syncthingtray.json:4]:
```suggestion
    "description": "Tray application for Syncthing",
    "homepage": "https://github.com/Martchus/syncthingtray",
```

File: bucket/syncthingtray.json [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/Martchus/syncthingtray/releases/download/v1.1.1/syncthingtray-1.1.1-i686-w64-mingw32.exe.zip",
+            "hash": "802866f997c0bf03cc70f8969ce402036e6831ce94d7d63cb471e7be73fce98a"
+        },
+        "64bit": {
+            "url": "https://github.com/Martchus/syncthingtray/releases/download/v1.1.1/syncthingtray-1.1.1-x86_64-w64-mingw32.exe.zip",
+            "hash": "85c16af32fcafacc7e8d37609f0bdfb3b038e922728d51577083f47f9bbb3565"
+        }
```
Comment (Ash258) [inline bucket/syncthingtray.json:17]:
```suggestion
        "64bit": {
            "url": "https://github.com/Martchus/syncthingtray/releases/download/v1.1.1/syncthingtray-1.1.1-x86_64-w64-mingw32.exe.zip",
            "hash": "85c16af32fcafacc7e8d37609f0bdfb3b038e922728d51577083f47f9bbb3565"
        },
        "32bit": {
            "url": "https://github.com/Martchus/syncthingtray/releases/download/v1.1.1/syncthingtray-1.1.1-i686-w64-mingw32.exe.zip",
            "hash": "802866f997c0bf03cc70f8969ce402036e6831ce94d7d63cb471e7be73fce98a"
        }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/syncthingtray.json (hunk 2 above)
Comment (yan12125) [reply bucket/syncthingtray.json:28]: I guess this means `checkver` should come before `autoupdate` and simply "github" is enough? Done with the latest version of this PR.

File: bucket/syncthingtray.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "1.1.1",
+    "description": "Tray application for Syncthing",
+    "homepage": "https://github.com/Martchus/syncthingtray",
+    "license": {
+        "identifier": "GPL-2.0,...",
```
Comment (Ash258) [inline bucket/syncthingtray.json:6]:
```suggestion
        "identifier": "GPL-2.0-only,...",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Why do you prefer QT5 over QT6?

A

**Comment by yan12125** [comment]
> Why do you prefer QT5 over QT6?

Sorry my oversight. I created this JSON file some time ago for an older version. There was no Qt 6 binaries then, and I didn't check if new versions are compiled with Qt 6 or not. I switched to Qt 6 as well as made the license more accurate in the latest version of this PR.

**Review by rasa — APPROVED** [review]

**Comment by Ash258** [comment]
@rasa: You really merged untested qt6 version??  Why do you merge random PRs, which were not accepted??

**Comment by yan12125** [comment]
For what it's worth, I have been using the Qt 6 version since you reminded me of that 3 days ago, and all features I use (e.g., launching the built-in syncthing library and syncing with other peers, checking the status of devices/folders/recent changes) work fine. I'm also willing to provide further test results if you point me what is needed.

**Comment by rasa** [comment]
> @rasa: You really merged untested qt6 version?? Why do you merge random PRs, which were not accepted and have open discussions??

It appeared that the changes you requested had been implemented. Also, @yan12125 claimed he did test the qt6 version. If you would prefer to handle all merge requests, that would be fine.

**Comment by AntonOks** [comment]
> 
> 
> > @rasa: You really merged untested qt6 version?? Why do you merge random PRs, which were not accepted and have open discussions??
> 
> It appeared that the changes you requested had been implemented. Also, @yan12125 claimed he did test the qt6 version. If you would prefer to handle all merge requests, that would be fine.

THANKS @rasa

**Pull request merged**